DataPrivacy.html

```diff
@@ -54,10 +55,10 @@ <h1>Privacy Policy for Gammapy Websites</h1>
             <p><b>Important:</b> all data collected is strictly anonymized, and we do not collect any personally identifiable
             information (PII).</p>
 
-            <h1>2. How We Collect Data</h1>
+            <h2>How We Collect Data?</h2>
```

Comment:
Should we make these titles at a different level to the top one? 
i.e. the red styled ones?

```suggestion
            <h2>How do we collect data?</h2>
```